cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ListPrivilegeTest.scala

```diff
@@ -53,45 +53,45 @@ WHERE role = 'my_role'
 
 List information about privileges, filtered by role, action and access and further refined by the name of the role.
 
-###assertion=show-one
+###assertion=show-two
```

Comment:
Do we want to keep the 2 above ones at all on the refcard if we want to steer users away from using the old SHOW PRIVILEGE?

Reply:
I think I'll move it to the top so I keep the ones without `AS COMMAND` together